modules/EvseManager/EvseManager.cpp

```diff
@@ -1329,13 +1315,7 @@ bool EvseManager::is_reserved() {
     return reserved;
 }
 
-bool EvseManager::get_hlc_enabled() {
-    Everest::scoped_lock_timeout lock(hlc_mutex, Everest::MutexDescription::EVSE_get_hlc_enabled);
-    return hlc_enabled;
-}
-
 bool EvseManager::get_hlc_waiting_for_auth_pnc() {
```

Comment:
Is this get function necessary? `hlc_waiting_for_auth_pnc` is public

Reply:
Looks like `hlc_waiting_for_auth_pnc` is private